### PR TITLE
Readme copyedits

### PR DESCRIPTION
**Pull request by lloydde**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,18 +31,18 @@ For additional resources, you can visit the
...
 remainder are compute nodes (CNs) which run instances (virtual machines).
 
 SDC features:
 
 - SmartOS zones provides high performance container virtualization. KVM support
   on top of zones means secure full Linux and Windows guest OS support.
-- RESTful API and CLI tooling for customer self-service
-- Complete operations portal (web GUI)
+- RESTful API and CLI tooling for customer self-service.
+- Complete operator portal (web app).
```
Comment (trentm) [inline README.md]: These aren't sentences. I'm no grammarian, but doesn't that mean we shouldn't use a period?

File: README.md [hunk 2]
```diff
@@ -281,47 +198,42 @@ SDC is composed of several pre-built components:
...
 builds (see the preceding sections) via the [sdc-headnode
-repository](https://github.com/joyent/sdc-headnode). Built components are typically
-stored in a [Manta object store](https://github.com/joyent/manta), e.g.
-[Joyent's public Manta](https://www.joyent.com/products/manta), and pulled from
-there. For example, Joyent's core builds push to
+repository](https://github.com/joyent/sdc-headnode). The built components are
+typically stored in a [Manta object store](https://github.com/joyent/manta),
+e.g. [Joyent's public Manta](https://www.joyent.com/products/manta),
+and pulled from there. For example, Joyent's master builds push to
 `/Joyent_Dev/public/builds` in Joyent's public Manta in us-east-1
```
Comment (trentm) [inline README.md]: That should stay "core", not "master". We do builds of branches other than "master".  Or if you meant "master" as other than the branch name, I suppose that is confusing overloading of "master". :)

**Comment by trentm** [comment]
@lloydde All looks good, with one exception: The dropping of steps 4-8 of the COAL getting started.  I think there is good value in having that full process "from download to running COAL" up front on the first page rather than spread across two documents. Someone scanning quickly could easily miss the jump to coal-setup.md, IMO. I realize that means redundancy.

If contentious perhaps we could separate that change to a separate ticket to get your other copy edits in?

File: README.md (hunk 1 above)
Comment (lloydde) [inline README.md]: Thanks @trentm. I misinterpreted the grammar guide I decided to go with.

File: README.md (hunk 2 above)
Comment (lloydde) [inline README.md]: will remove core in updated pull

**Comment by lloydde** [comment]
Continued in joyent/sdc#48.

**Pull request closed without merging**